pkgs/applications/misc/gcstar/default.nix

```diff
@@ -0,0 +1,70 @@
+{ stdenv
+, fetchFromGitLab
+, perlPackages
+, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gcstar";
+  version = "1.7.2";
+
+  src = fetchFromGitLab {
+    owner = "Kerenoc";
+    repo = "GCstar";
+    rev = "v${version}";
+    sha256 = "1vqfff33sssvlvsva1dflggmwl00j5p64sn1669f9wrbvjkxgpv4";
+  };
+
+  buildInputs = [ wrapGAppsHook ] ++ (with perlPackages; [
```

Comment:
`wrapGAppsHook` should go to `nativeBuildInputs`, though.

Reply:
Fixed. Thanks